diff --git a/dist/module.mjs b/dist/module.mjs
index f0ca0857af7dd14fa64c9a92ca5a1ff093891a9e..3f91425708c285b039db5ecfc6348e54ed6d25cd 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -42,7 +42,7 @@ const module = defineNuxtModule({
         extensions: []
       });
     });
-    await installModule("@nuxtjs/tailwindcss");
+    // await installModule("@nuxtjs/tailwindcss");
     await installModule("@nuxtjs/color-mode");
     try {
       readdirSync(resolve(COMPONENT_DIR_PATH)).forEach(async (dir) => {
